assignment2

efrenltt24 opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
I’m adding  SQL query and codes  to retrieve data from a database
## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
in this case there were different approaches that could give me same results. i used  the fastest and easiest to implement

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
writing sql code can be very challenging especially if syntax is not properly done.  had several errors and had to do lots of trial and error
## How were these changes tested?
every time i changed my coding, i run  it until  the outcomes were positive. I had to do this several times.
## A reference to a related issue in your repository (if applicable)

## Checklist
- [ x] I can confirm that my changes are working as intended

efrenltt24 committed:
assignment2-finished

efantinatti reviewed (approved):
Dear efren Tumialan tenorio

Session 1:
1 - Partial. Missing relationship types.

Session 2:
2 - OK.

Session 3:
3 - Partial.
-- Cross Join - Q1: CROSS JOIN is twice. Results are not correct. Consider leaving CROSS JOIN only with `customers` table.

Final:
4 - Check the aforementioned points prior to merging your feature branch 'assignment-two' to 'main'.

efrenltt24 committed:
final fix to assignment2

efrenltt24 commented:
Dear Ernani, I have added the relationship types on session1. I have also fixed session 3 q1 and used only 1 cross join with the customer table. Let me know if it works now. Thank you

Efren

________________________________
From: Ernani Fantinatti ***@***.***>
Sent: December 24, 2024 11:06 AM
To: efrenltt24/sql ***@***.***>
Cc: efrenltt24 ***@***.***>; Author ***@***.***>
Subject: Re: [efrenltt24/sql] assignment2 (PR #2)


@efantinatti approved this pull request.

Dear efren Tumialan tenorio

Session 1:
1 - Partial. Missing relationship types.

Session 2:
2 - OK.

Session 3:
3 - Partial.
-- Cross Join - Q1: CROSS JOIN is twice. Results are not correct. Consider leaving CROSS JOIN only with customers table.

Final:
4 - Check the aforementioned points prior to merging your feature branch 'assignment-two' to 'main'.

—
Reply to this email directly, view it on GitHub<https://github.com/efrenltt24/sql/pull/2#pullrequestreview-2522092327>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BM5RKJV45JATTSVIMO3MJL32HGBADAVCNFSM6AAAAABUBM4EOGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMRSGA4TEMZSG4>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
